examples/aggregator/module/rust/tests/integration_test.rs

```diff
@@ -49,7 +49,11 @@ async fn test_aggregator() {
         "oak_main",
         ConfigMap {
             items: hashmap! {
-                "config".to_string() => br#"grpc_server_listen_address = "[::]:8080""#.iter().cloned().collect()
+                "config".to_string() => br#"
+                    grpc_server_listen_address = "[::]:8080"
+                    backend_server_address = "https://localhost:8888"
+                    aggregator_module_hash = "49480d98a87569ba39b0b29bc739c9f863502de73c964a8e54f6d720045de2c3"
```

Comment:
The hash is repreated in many places. Perhaps add documentation somewhere to explain how to get the new hash and all the different places where the hash needs to be updated if the aggregator example needs to be rebuilt.

Reply:
Done